### DISCUSSION TITLE
Flask Issues All Of A Sudden

### DISCUSSION THREAD

**Discussion by adambotfd** [vercel/community#4305] [Help]
### Summary

Yesterday my flask app was working fine, i woke up this morning (there has been no changes or deployments since it was working) and the logs are just saying this whenever anyone visits the site:

`[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': cannot import name 'url_quote' from 'werkzeug.urls' (/var/task/werkzeug/urls.py)
Traceback (most recent call last):`

however the thing is i am not using any `url_quote` or `werkzeug.urls`

### Example

https://guardianapi.vercel.app/botlist/10

- yesterday this url used to return a json list however now returns a status code 500

### Steps to Reproduce

No idea

**Comment by june9713** [comment]
I have same issues too.

**Reply by adambotfd** [reply]
glad its not just me

**Reply by june9713** [reply]
https://youtu.be/pAyKJAtDNCw?si=SW7GXD37GnDlWPN1

**Comment by AkashiCoin** [comment]
I have same issues too.

---

I try to use `werkzeug==2.2.2` in `requirements.txt` and it work.
ps: flask==2.2.2

**Reply by june9713** [reply]
was it not 2.2.2?

**Comment by spookyuser** [comment]
samesies 😭

For me it's Unable to import module 'vc__handler__python': No module named 'numba'

**Comment by lunasupps** [comment]
I have the same issue. No clue why

[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': cannot import name 'url_quote' from 'werkzeug.urls' (/var/task/werkzeug/urls.py)
Traceback (most recent call last):

**Reply by spookyuser** [reply]
I forgot what the exact reason was for me, but it did turn out to be my fault, i think i wasn't locking my packages correctly with pipenv or was locking the wrong category or something

**Reply by spookyuser** [reply]
But also i just stopped using vercel for python shortly afterward bc it's impossible to debug

**Reply by lunasupps** [reply]
Haha true
